.github/workflows/packaging-algo.yml

```diff
@@ -0,0 +1,29 @@
+name: release
+# generating and testing package artefacts from the main branch
+
+on:
+  workflow_dispatch:
+
+jobs:
+  packaging-algo:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        ref: main
+    - name: Algo name
+      id: name
+      run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+    - name: Build
+      env:
+        release_version: ${{ steps.name.outputs.sha_short }}
+      run: |
+        echo $release_version
```

Comment:
I am not familiar with GitHub actions. When I do it locally, sometimes there are pycache files generated after local testing. So I have one extra line in my local script before the packaging:
```
find . | grep -E "(/__pycache__$|\.pyc$|\.pyo$)" | xargs rm -rf
```

Reply:
it doesn't look like an issue as we have the .gitignore to remove these when pushing to github. the tar command logs suggest the same. https://github.com/Project-MONAI/research-contributions/actions/runs/4736101445/jobs/8407194485